src/pooling.jl

```diff
@@ -166,7 +180,32 @@ function meanpool(x, k::NTuple{N, Integer}; pad=0, stride=k) where N
 end
 
 
-for pool in [:maxpool, :meanpool]
+"""
+    lpnormpool(x, p::Number, k::NTuple{N, Integer}; pad=0, stride=k)
+
+Perform Lp pool operation with value of the Lp norm `p` and window size `k` on input tensor `x`, also known as LPPool in pytorch.
+This pooling operator from [Learned-Norm Pooling for Deep Feedforward and Recurrent Neural Networks](https://arxiv.org/abs/1311.1780).
+
+* `x` and `k`: Usually, ndim(x) ∈ [3, 5], length(k) ∈ [1, 3], s.t. ndim(x) == length(k) + 2
+* `p` is restricted to (0, Inf). Out-of-range `p` leads to undefined behavior.
```

Comment:
It's an error not undefined, right? And `(0, Inf)` could mean a tuple, clearer as code?
```suggestion
* `p` is restricted to `0 < p < Inf`.
```

Reply:
Thanks for your patient review! I have updated this in new commit.